doc/en/components/charts/chart-type-overview.md

```diff
@@ -1,32 +1,212 @@
+
+Learn more about our [line charts](chart-types-line.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+### $PlatformShort$ Doughnut Chart
+
+The $PlatformShort$ Doughnut Chart, or Doughnut Graph, is a variant of a Pie Chart, proportionally illustrating the occurrences of a variable in a circle to represents parts of a whole. The doughnut chart has a circular opening at the center of the pie chart, where a title or category explanation can be displayed. Doughnut charts can support multiple concentric rings, with built-in support for visualizing hierarchical data.
+
+Learn more about our [doughnut charts](../doughnut-chart.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+### $PlatformShort$ Area Chart
+
+Learn more about our [area charts](chart-types-area.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+### Sparkline Chart 
+
+A Sparkline Chart, or Sparkline Graph is a type of category graph intended for rendering within a small-scale layout such as within a grid cell, or anywhere a word-sized visualization is needed to tell a data story. Like other $PlatformShort$ chart types, the Sparkline Chart has several visual elements and corresponding features that can be configured and customized such as the chart type, markers, ranges, trendlines, unknown value plotting, and tooltips. Sparkline charts can render as a Line Chart, Area Chart, Column Chart or Win / Loss Chart. The difference between the full-sized chart equivalent to the Spark-chart, is the Y-Axis (left side labels) and X-Axis (bottom labels) are not visible. Learn more about our [sparkline charts](../sparkline.md).
+
+### Bubble chart 
+
+A Bubble Chart, or Bubble Graph, is used to show data comprising of three numeric values. Two of the values are plotted as an intersecting point using a Cartesian (X, Y) coordinate system, and the third value is rendered as the diameter size of the point. This gives the Bubble Chart its name - a visualization of varying sized bubbles along the X and Y coordinates of the plot. The $PlatformShort$ Bubble Chart is used to show relationships of data correlations with the data value differences rendered by size. You can also use a fourth data dimension, typically color, to further differentiate the values in your Bubble chart. Learn more about our [bubble charts](chart-types-scatter.md#$PlatformShort$-Scatter-Bubble-Chart).
+
```

Comment:
I've made a generic Bubble chart and we can use the following data for the sample:

[annual-working-hours-vs-gdp-per-capita-pwt.xlsx](https://github.com/IgniteUI/igniteui-xplat-docs/files/5886035/annual-working-hours-vs-gdp-per-capita-pwt.xlsx)

It can be seen here in action: https://ourworldindata.org/grapher/annual-working-hours-vs-gdp-per-capita-pwt?tab=chart&stackMode=absolute&time=2017..latest&country=&region=World